src/lib/CrsfProtocol/crsf_protocol.h

```diff
@@ -129,6 +130,7 @@ enum {
 enum {
     CRSF_FRAME_GPS_PAYLOAD_SIZE = 15,
     CRSF_FRAME_VARIO_PAYLOAD_SIZE = 2,
+    CRSF_FRAME_BARO_ALTITUDE_PAYLOAD_SIZE = 3,
```

Comment:
I believe the size should be `2`, just like the `VARIO` frame.

Reply:
Appears to me that it is 2 bytes for the altitude and 1 for vario, so 3 sounds right and also is the size in the betaflight PR:
https://github.com/betaflight/betaflight/pull/11070/files#diff-5b4ded67d6b41332a7188e3100bec940e517ea32550f01e25709b13f86eabc77R313

Reply:
@CapnBry i can add it, no problem, but I’m still not convinced, really.

Reply:
@raphaelcoeffic I'm also not convinced with the 3 bytes. We could start with 2 bytes and change it later if it really gets merged to bf.

Reply:
size changed to two bytes